policykit/slackintegration/views.py

```diff
@@ -143,6 +102,76 @@ def oauth(request):
     response = redirect('/login?success=true')
     return response
 
+@csrf_exempt
+def initSlackCommunity(request):
```

Comment:
one q I have is how much of all this is slack-specific versus something we can do for all integrations at once.